### ISSUE TITLE
add emoji to menu

### ISSUE THREAD

**Issue by progserega** [vector-im/riot-web#6760]
Please, add emoji-tab to 'stickers' popup window. 
Add emoji in riot only after type ':' - is not simple.
![imoji](https://user-images.githubusercontent.com/1297163/40207271-0ed9fe1c-5a77-11e8-8eaf-d82f30811be0.png)

**Comment by tulir** [comment]
Related/duplicate of #1107

**Comment by t3chguy** [comment]
Superseded by #1107 indeed, emoji picker through Scalar would become messy as then it'd need a way to *securely* interact with UI elements such as the composer
